### PR TITLE
[AR-44] FE: Our Mission Section Responsiveness

### PR DESCRIPTION
**Pull request by wagbubu**
I removed ContentLink Component and just re-use ButtonStyledLink instead

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -67,6 +67,7 @@ module.exports = {
         md: "1rem", //16px
         lg: "1.125rem", //18px
         xl: "1.25rem", //20px
+        "2.5xl": "1.75rem",
```
Comment (idiglove) [inline tailwind.config.js]: just use xl2 in this situation

**Review by idiglove — CHANGES_REQUESTED** [review]
thanks, 1 small comment

File: tailwind.config.js (hunk 1 above)
Comment (wagbubu) [reply tailwind.config.js]: oh, I forgot to remove it, it's unused 😅

**Review by idiglove — APPROVED** [review]
awesome thanks ⭐

**Pull request merged**